### ISSUE TITLE
Distinguish between frameworks that use custom UI libraries and frameworks that support multiple UI libraries

### ISSUE THREAD

**Issue by FeldrinH** [lxsmnsyc/awesome-islands#1]
Currently the Standalone/Metaframework category includes both frameworks that support multiple existing UI libraries (Astro, Iles) and frameworks that provide their own custom UI rendering solution (Qwik, Marko, Slinkity). It feels like these should really be two distinct categories.

**Comment by lxsmnsyc** [comment]
Yeah you're right, but it felt weird initially because Astro also has its own DSL, so it felt like it's the same level as Marko, then there's also Qwik which supports React. I'm still in dilemma.
